src/app/frontend/replicasetdetail/updatereplicas_controller.js

```diff
@@ -0,0 +1,77 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Controller for the update replica set dialog.
+ *
+ * @final
+ */
+export default class UpdateReplicasDialogController {
+  /**
+   * @param {!md.$dialog} $mdDialog
+   * @param {!angular.$log} $log
+   * @param {!function(number, function(!backendApi.ReplicaSetDetail),
+   * function(!angular.$http.Response)=)} updateReplicasFn
+   * @ngInject
+   */
+  constructor($mdDialog, $log, updateReplicasFn) {
+    /** @export {number} */
+    this.replicas = 1;
```

Comment:
I'd actually say that this should be current value. To see what you're changing from. Or, at least, have empty input here  and display current number of replicas in the description. 

The reason is I don't think that `1` is any better than `7` or `12312`.